cdap-security/src/test/java/co/cask/cdap/security/authorization/ExceptionAuthorizationEnforcer.java

```diff
@@ -43,7 +44,7 @@ public void enforce(EntityId entity, Principal principal, Set<Action> actions) t
 
   @Override
   public Set<? extends EntityId> isVisible(Set<? extends EntityId> entityIds, Principal principal) throws Exception {
-    throw new UnsupportedOperationException("This method needs to be implemented!");
+    return Collections.emptySet();
```

Comment:
This class is only meant to throw Exception when `enforce` is called. The other methods seems unused. The `createFilter` method returns null instead of throw exception, which I think we can just return an empty list.